airbyte-integrations/connectors/source-s3/source_s3/source_files_abstract/spec.py

```diff
@@ -111,17 +111,18 @@ def check_provider_added(schema: dict) -> None:
     def resolve_refs(schema: dict) -> dict:
```

Comment:
This seems to be duplicated code as well. I'm wondering if we should re-use the implementation in our cdk utils package